build.gradle.kts

```diff
@@ -118,7 +118,7 @@ subprojects {
 
 allprojects {
   group = "org.stellar.anchor-sdk"
-  version = "1.2.2"
+  version = "1.2.4"
```

Comment:
Should we release 1.2.3?

Reply:
There is a release for this version already
https://github.com/stellar/java-stellar-anchor-sdk/releases/tag/release-1.2.3